.github/workflows/Excercise7.yml

```diff
@@ -2,11 +2,30 @@ name: Continuous Integration
 on: [push]                      # Will make the workflow run every time you push to any branch
 
 jobs:
-  build:
+  test:
     name: Build and test
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - run: docker build --target test --tag todo_apptest .
     - run: docker run --env-file .env.test todo_apptest todo_app/test
-    - run: docker run -e TRELLOAPIKEY=${{ secrets.TRELLOAPIKEY }} -e TRELLOAPITOKEN=${{ secrets.TRELLOAPITOKEN }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} todo_apptest todo_app/end_to_end_test
\ No newline at end of file
+    - run: docker run -e TRELLOAPIKEY=${{ secrets.TRELLOAPIKEY }} -e TRELLOAPITOKEN=${{ secrets.TRELLOAPITOKEN }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} todo_apptest todo_app/end_to_end_test
+  build:
+    name: Deploy
+    needs: test
+    if: ${{ github.ref == 'refs/heads/Exercise_8' }}
```

Comment:
Don't forget to change this to the main branch when merging. 

And you can also change it if you are working on another branch and want to test those changes on Heroku.